src/AnnotationThread.js

```diff
@@ -454,11 +454,9 @@ class AnnotationThread extends EventEmitter {
      * @return {void}
      */
     unbindDOMListeners() {
-        if (!this.element) {
-            return;
+        if (this.element) {
```

Comment:
It looks like we're using guard clauses in some files and wrapping expressions in the condition in others. This style seems like it may be preferable due to inheritance interactions. Should we standardize?